dmg/default/ArangoDB3e-CLI.app/Contents/Resources/arangodb

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+realpath() {
+  OURPWD=$PWD
+  cd "$(dirname "$1")"
+  LINK=$(readlink "$(basename "$1")")
+  while [ "$LINK" ]; do
+    cd "$(dirname "$LINK")"
+    LINK=$(readlink "$(basename "$1")")
+  done
+  REALPATH="$PWD/$(basename "$1")"
+  cd "$OURPWD"
+  echo "$REALPATH"
+}
+me=$(realpath $0)
+program=$(basename $me)
+installation=$(dirname $me)
+executable=$installation/opt/arangodb/bin/$program
+if [ -x $executable ] ; then
+  exec $executable --server.arangod=$installation/opt/arangodb/sbin/arangod --server.js-dir=$installation/opt/arangodb/share/arangodb3/js "$@"
+fi
+echo "Could not find executable!"
```

Comment:
should we print $executable here?

Reply:
See https://github.com/arangodb/oskar/pull/212/commits/786d8c83d11e5a15b3860c6feefae7a4dfa57631.